src/redux/actions/examples.js

```diff
@@ -80,3 +80,19 @@ export const downloadResults = () => async (dispatch) => {
     console.error(`Error downloading results: ${error.message}`);
   }
 };
+
+/*
+export const downloadPartMC = () => async (dispatch) => {
+  try {
+    const url = await fetchPartMC();
+    const link = document.createElement("a");
+    link.href = url;
+    link.download = "PartmcResult.zip";
+    document.body.appendChild(link);
+    link.click();
+    link.parentNode.removeChild(link);
+  } catch (error) {
+    console.error(`Error downloading results: ${error.message}`);
+  }
+};
+*/
```

Comment:
```suggestion
```

This function is not needed. We discourage keeping around commented code. Either it isn't needed or we need to create an issue to include it at a later date. In this case, it isn't needed